srcpkgs/ardour/template

```diff
@@ -31,14 +28,19 @@ case "$XBPS_TARGET_MACHINE" in
 	*)        broken="Don't know build target" ;;
 esac
 
+do_fetch() {
+	git clone --branch ${version} --depth=1 \
+	 git://github.com/Ardour/ardour ${pkgname}-${version}
+}
```

Comment:
This needs a comment on why, though it is kinda acceptable, given that they delete old tarballs :(

Maybe we could hardcode the commit for the given tag? Instead of pulling the tag directly. Not sure that's supported or if it'd make the clone too big.

Reply:
>Maybe we could hardcode the commit for the given tag? Instead of pulling the tag directly. Not sure that's supported or if it'd make the clone too big.

Why? Whats wrong with using the tag directly?